cypresse2e/cypress/integration/e2eTests/joinNow/joinNow.spec.js

```diff
@@ -41,6 +41,48 @@ describe('FightPandemics Sign Up Page', () => {
 			validEmailRequired.contains('small', errorMessages.invalidEmail);
 		});
 
+		it('Entering @ twice or more in email triggers error', () => {
+			var emailField = joinNow.getEmailField();
```

Comment:
@JayashreeSamantasinghar 
Here we have some repeatable code. I would refactor it as in the https://github.com/FightPandemics/FightPandemics/blob/staging/cypresse2e/cypress/integration/e2eTests/header/header.spec.js
Where we have a function, pass parameters.
Similar example: https://github.com/FightPandemics/FightPandemics/blob/staging/cypresse2e/cypress/integration/e2eTests/joinNow/joinNow.spec.js